ios/RNWifi.m

```diff
@@ -106,7 +106,10 @@ + (BOOL)requiresMainQueueSetup
                   resolver:(RCTPromiseResolveBlock)resolve
                   rejecter:(RCTPromiseRejectBlock)reject) {
     // Prevent NEHotspotConfigurationManager error when connecting to an already connected network
-    if ([ssid isEqualToString:[self getWifiSSID]]) resolve(nil);
+    if ([ssid isEqualToString:[self getWifiSSID]]){
+        reject(@"nehotspot_error", @"You are trying to connect to a network you are already connected to", nil);
```

Comment:
Actually just resolve and return in this case.

Reply:
@eliaslecomte, is right! To prevent _false negatives_ and confusing the user, it makes sense to just resolve if you are already connected to the network.

Reply:
Agree